server/test/controllers/attainable.test.ts

```diff
@@ -0,0 +1,293 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import supertest from 'supertest';
+
+import { app } from '../../src/app';
```

Comment:
I think this counts as an "other" import in our [code style](https://github.com/aalto-grades/base-repository/wiki/Code-style#import-order) and should be grouped with the type imports below.